packages/adminpanel-v2/src/components/customerpage/customerlist/customers/Customers.svelte

```diff
@@ -1,13 +1,13 @@
 <script lang="ts">
 	import type { TCustomer } from "$lib/types";
+	import { customers } from "../../../../stores/CustomerStore";
     import Customer from "./customer/Customer.svelte";
-    export let customers : TCustomer[]
     export let selectCustomer : Function
 </script>
 
 <div class="left-component">
     <ul class="customer-list">
-        {#each customers as customer}
+        {#each $customers.customers as customer}
```

Comment:
What does it mean when you use it inline inside a loop, does it have any specific cases then or do you always call it with the $ sign when you use a reactive variable?

Reply:
In this case $customers refers to the store. The $ sign is generally reserved to store referencing, except for when used to declare reactive variables. When referring to a store, $ must be used for store variables, but not when referring to store functions.

Reply:
aaaah okay, nice. That makes sense too